app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,70 @@
+class CommentsController < ApplicationController
+
+  before_action :authenticate_user!
+  before_action(:find_comment, {only: [:show, :edit, :update, :destroy]})
+  before_action :authorize_comment, only: [:edit, :update, :destroy]
```

Comment:
You can just write 
before_action :find_comment, only: [:show, :edit, :update, :destroy]
to make them consistent.